slices/ca-certificates-java.yaml

```diff
@@ -0,0 +1,33 @@
+package: ca-certificates-java
+
+essential:
+  - ca-certificates-java_copyright
+
+slices:
+  bins:
```

Comment:
Should this be named `data` in similarity with the one from 22.04?

Reply:
This slice provides a runnable jar file that is used to populate Java keystore. I can split out data slice that will  create /etc/ssl/certs/java/ in order to keep the name.